auth-web/src/components/pay/PaySystemAlert.vue

```diff
@@ -45,7 +45,7 @@ export default class PaySystemAlert extends Vue {
     this.statusStore.fetchPaySystemStatus()
   }
   get isPaySystemDown () {
-    return !this.getBoolean(this.statusStore.paySystemStatus.currentStatus)
+    return true
```

Comment:
Sure about this change? :)

Reply:
Whups. Removing and resubmitting. Thanks for catching.